### DISCUSSION TITLE
no option to create role from api

### DISCUSSION THREAD

**Discussion by maya-harel** [apache/superset#18969] [Q&A / Help]
Hi
I want to integrate Superset into my application. To do this I want to create a role for each schema (My data is on AWS Athena), and then assign it to a group of users 
Up until now, I have been working with the API (https://superset.apache.org/docs/api/) but I am unable to create/update/delete roles using the API
Is there any way to do this programmatically?

**Comment by srinify** [comment]
Hi @maya-harel I'm moving this over to Discussions since this isn't a bug :]

**Comment by xunfeng1980** [comment]
@maya-harel Hi, do you have any ideas?
